framework/routing/__init__.py

```diff
@@ -392,12 +413,15 @@ def __init__(self, template_name, renderer=None, error_renderer=None,
                      to add to data from view function
         :param detect_render_nested: Auto-detect renderers for nested
             templates?
+        :param safe: Boolean: turn on markup-safe escaping?
```

Comment:
Now that `Status` uses the parameter `trust`, is there any reason not to go with that name here as well?